.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "Questions/Help/Support"
+about: Resources to get help and support about NumPy.
+
+---
+
+## Questions and Help
+
+### Please note that this issue tracker is not a help form and this issue will
+be closed.
```

Comment:
Instead of using a template here at all, you can make this be a template, like numba does

https://github.com/numba/numba/blob/master/.github/ISSUE_TEMPLATE/config.yml

Reply:
That does looks better, I'll update, thanks!